packages/cli/src/commands/doctor.ts

```diff
@@ -1,5 +1,63 @@
+import Chalk from 'chalk';
 import { Config } from '@react-native-community/cli-types';
+import getAccount from '../actions/getAccount';
+import log from '../helpers/log';
+import file from '../helpers/file';
+import getConfig from '../actions/getConfig';
+import prompt from '../helpers/prompt';
+import firebase from '../helpers/firebase';
+import { getAndroidApp } from '../actions/getApp';
 
 export default async function doctorCommand(args: string[], reactNativeConfig: Config) {
-  console.log('todo');
+  const [androidProjectConfig, iosProjectConfig] = getConfig(reactNativeConfig);
+
+  const account = await getAccount(true);
+  if (account) {
+    var firebaseProject = await prompt.selectFirebaseProject(account);
+    if (!firebaseProject)
+      log.warn(
+        `No Firebase projects exist for user ${Chalk.cyanBright(
+          `[${account.user.email}], Firebase information will not be available.`,
+        )}`,
+      );
+    else {
+      var projectDetail = await firebase
+        .api(account)
+        .management.getProject(firebaseProject.projectId, { android: true, ios: true, web: false });
+      log.info('# Firebase project information');
+      log.info(`Display name: ${projectDetail.displayName}`);
+      log.info(`Project ID: ${projectDetail.projectId}`);
+      log.info(`Project number: ${projectDetail.projectNumber}`);
+    }
+  } else {
+    log.warn('Not logged into Firebase, Firebase information will not be available.');
+  }
+
+  log.info('# Firebase Android information');
+  log.info(`Package name: ${androidProjectConfig.packageName}`);
+  if (projectDetail) {
+    var androidApp = getAndroidApp(projectDetail, androidProjectConfig.packageName);
+    if (androidApp) {
+      log.info(`Display name: ${androidApp.displayName}`);
+      log.info(`AppId: ${androidApp.appId}`);
+    }
+  }
+  const androidGoogleServicesFile = await file.readAndroidGoogleServices(androidProjectConfig);
+  if (androidGoogleServicesFile) {
+    log.success('A "google-services.json" has been found.');
+  } else {
+    log.warn(
+      'No "google-services.json" file has been found, Firebase for Android has not been setup.',
+    );
+  }
+
+  log.info('# iOS');
+  const iosGoogleServicesFile = await file.readIosGoogleServices(iosProjectConfig);
+  if (iosGoogleServicesFile) {
+    log.success('A "GoogleService-Info.plist" has been found.');
+  } else {
+    log.warn(
+      'No "GoogleService-Info.plist" file has been found, Firebase for iOS has not been setup.',
+    );
+  }
```

Comment:
Can we have the doctor command print the following also, useful for issue triage;

 - `@react-native-firebase/*` NPM packages being used & their versions (could read from package.json)
 - `react-native` version being used (could read from package.json)

Reply:
cc @mikehardy for additional thoughts

Reply:
Very interesting... I like the SHA check, should check debug and release somehow but I'm not sure how feasible that is. Thinking of where all the issue traffic comes from, I think missing an auth listener while you are using the auth package is a red flag especially if a phone auth confirm code is directly checked. Integrating react-native-splash-screen as LAUNCHER activity instead of MainActivity is a red flag. So much messaging stuff - putting a background handler outside of App.js/index.js is a warning ? Modules in package.json that are not in Podfile.lock (pod install failure).